bin/validate_dn_format.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import os
+import re
+
+regex = re.compile(r"^(/(?:DC|OU|C|ST|L|O)=[^/=]+)*(/CN=[^/=].+)$")
+regexCN = re.compile(r"(?<=CN=).*")
+
+errorLog = []
+
+def validate(dir):
+    for root, subdirs, files in os.walk(dir):
+        for file in files:
+            if file.endswith('.lsc'):
```

Comment:
(Sorry just happened to notice this PR as i was doing some other work in this repo)

Alternatively, since we know all the lsc files are under `vomsdir/*/*.lsc`

```suggestion
    for file in glob.glob(dir + "/*/*.lsc"):
```